### PR TITLE
Add BBE for websocket query param

### PR DESCRIPTION
**Pull request by MohamedSabthar**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/websocket-query-parameter/websocket_query_parameter.bal [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import ballerina/websocket;
+
+service /chat on new websocket:Listener(9090) {
+
+    // The `userName` parameter in the resource method is treated as a query parameter,
+    // which is extracted from the request URI. For example, invoking this service with
+    // the URI `ws://localhost:9090/chat?userName=xyz` passes `xyz` as the value
+    // to the `userName` parameter defined in the resource method.
```
Comment (NipunaRanasinghe) [inline examples/websocket-query-parameter/websocket_query_parameter.bal:8]: Let's stick to real world values, whenever possible
```suggestion
    // The `userName` parameter in the resource method is treated as a query parameter,
    // which is extracted from the request URI. For example, invoking this service with
    // the URI `ws://localhost:9090/chat?userName=John` passes `John` as the value
    // to the `userName` parameter defined in the resource method.
```

File: examples/websocket-query-parameter/websocket_query_parameter.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# WebSocket service - Query parameter
```
[the remark this answers is not in the document]
Comment (MohamedSabthar) [reply examples/websocket-query-parameter/websocket_query_parameter.md:1]: Followed the same as http

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Comment by NipunaRanasinghe** [comment]
@MohamedSabthar shall we look into the BBE test failures?

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**